eq-author-api/README.md

```diff
@@ -33,52 +52,144 @@ In most cases sensible defaults have been selected.
 | `NODE_ENV`              | Sets the environment the code is running in                                        | No       |
 | `ENABLE_IMPORT`         | When enabled it exposes a post endpoint for importing questionnaires               | No       |
 
-## Run using Docker
+## Running with Docker Compose
 
 To build and run the Author GraphQL API inside a docker container, ensure that
 Docker is installed for your platform, navigate to the project directory, then run:
 
 Build the docker image (1st time run):
 
-```
+```bash
 docker-compose build
 ```
 
-```
+```bash
 docker-compose up
 ```
 
+Once the containers are running you should be able to navigate to <http://localhost:4000/graphiql> and begin exploring the eQ Author GraphQL API.
+
 Changes to the application should hot reload via `nodemon`.
 
+### Querying pages
+
+There is no concrete `Page` type in the GraphQL schema. Instead we use a `Page` interface, which other types implement e.g. `QuestionPage` and `InterstitialPage`.
+
+To query all pages, and request different fields depending on the type, use [inline fragments](http://graphql.org/learn/queries/#inline-fragments):
+
+```gql
+query {
+  getQuestionnaire(id: 1) {
+    questionnaire {
+      sections {
+        pages {
+          id
+
+          # inline fragment for `QuestionPage` type
+          ... on QuestionPage {
+            guidance
+            answers {
+              id
+              label
+            }
+          }
+
+          # For purposes of example only. `InterstitialPage` doesn't exist yet
+          ... on InterstitialPage {
+            # doesn't exist yet
+            someField
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+## Running tests
+
+`yarn test` will start a single run of unit and integration tests.
+
+`yarn test --watch` will start unit and integration tests in watch mode.
+
 ## Migrations
 
+### Creating a new migration
+
 `runQuestionnaireMigrations` middleware is responsible for updating the schema version and running any necessary migrations.
 
 - `yarn create-migration [name]` will create a new migration in the `/migrations` directory.
 - Add the created migration to the bottom of the `migrations/index.js` array.
 - `runQuestionnaireMigrations` will execute necessary migrations on every request providing the schema has not already been migrated.
 
-## DynamoDB
+### DB migration
```

Comment:
This is a regression

Reply:
Accidentaly included it from my other, older branch.

Reply:
Its removed now.